.circleci/config.yml

```diff
@@ -1,24 +1,24 @@
 # config file for circleci
-version: 2 
+version: 2
 jobs:
-  build:
-    docker:
-      - image: circleci/node:14.15.4
-    steps:
-      - checkout
-      - restore_cache:
-          key: livgust-macovidvaccines.com-{{ checksum "package.json" }}2
-      - run:
-          name: npm ci
-          command: npm ci
-      - run:
-          name: npm run build
-          command: npm run build
-      - run:
-          name: npm test
-          command: npm test
-      - save_cache:
-          paths:
-            - ~/.npm
-            - node_modules
-          key: livgust-macovidvaccines.com-{{ checksum "package.json" }}2
+    build:
```

Comment:
Why make all of these indenting changes?  It obscures the real diffs here.

Reply:
Without it, the prettier pre-commit hook fails. Part of the intent of this PR is to ensure prettier has been run before committing, so I had to run it and make sure all files comply.

If you think a file should be excluded from formatting let me know, and i can add it to the .prettierignore file.